engine/pom.xml

```diff
@@ -215,7 +215,7 @@
                             javax.el.*;version="[2.0.0,4.0.0)";resolution:=optional,
                             org.xml.sax.*;version="0",
                             org.jboss.logging.*;version="[3.1.0,4.0.0)",
-                            com.fasterxml.classmate.*;version="[${classmate.version},2.0.0)",
+                            com.fasterxml.classmate.*;version="[1.3,2.0.0)",
```

Comment:
What's the rationale behind that? Fixing it seems like a step back on first sight?

Reply:
Well, it's fixed for all the others so I would say it's a step forward for consistency :).

The rationale behing that is that Payara/GlassFish is still using 1.3.3 so upgrading to 1.3.4 makes the Payara/GlassFish CDI/OSGi tests fail.

I could use the build-helper-plugin to extract the major.minor version but I'm not sure it's worth it. And I'm not even sure we would want to require 1.4 if we upgrade to 1.4. That's why I did it like that.

Reply:
FWIW, when I will work on making the pom files consistent with JBoss parent/WF parent, the whole range will be parameterized and put in the `<properties>` part of the parent pom; they won't be hidden in engine.

That's how it's done in Search now and I think it's better: we define the range we want but it's still very visible so we don't forget to update it if required.

This is for 6.1 though.

Reply:
Ok, that sounds good. On that consistency effort, I hope this won't mean we'll inherit from any JBoss or WF POM, right?

Reply:
> On that consistency effort, I hope this won't mean we'll inherit from any JBoss or WF POM, right?

Nope. That's the limit we set with Sanne.